metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -152,7 +153,11 @@
                    [conn db]
                    (let [conditions [`(>= :id ~start-index)
                                      `(< :id ~(+ start-index batch-size))]
-                         _ (info "Finding batch for provider" provider-id "from id >=" start-index " and id <" (+ start-index batch-size))
+                         _ (info (format "Finding batch for provider [%s] concept type [%s] from id >= %s and id < %s"
```

Comment:
Here are the queries my dashboard does:
sourcetype="prod_ngap_app_cmr-bootstrap-$cmr-env$" Finding batch | rex "Finding batch for provider (?&lt;provider&gt;[^ ]*) from" | chart count by provider

sourcetype="prod_ngap_app_cmr-bootstrap-$cmr-env$" Finding batch | chart count by host

The first one will break. But it's an easy fix.